src/components/GoodsList/GoodsList.scss

```diff
@@ -0,0 +1,13 @@
+.goods {
```

Comment:
filename must be the same as the block name

Reply:
good point